recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
-{% set version = "2.12.0" %}
+{% set version = "2.20.0" %}
```

Comment:
Was surprised about this seemingly huge jump, but that's what upstream [did](https://github.com/ray-project/ray/tags?after=ray-2.21.0) as well.

Reply:
Yes. Upstream moved to weekly releases. Maybe that jump was part of the signalling to end users about the change.